src/components/InteractiveMap/InteractiveMap.vue

```diff
@@ -54,49 +36,14 @@
       </v-container>
       <v-divider></v-divider>
       <v-container>
-        <v-card v-for="card in cards" :key="JSON.stringify(card)">
-          <v-toolbar dense color="primary" dark>
-            <v-toolbar-title class="body-2">{{ card.name }}</v-toolbar-title>
-            <v-spacer></v-spacer>
-            <v-icon>close</v-icon>
-          </v-toolbar>
-          <v-card-title primary-title class="py-2">
-            <v-layout justify-space-around>
-              <v-btn flat icon>
-                <v-icon>info</v-icon>
-              </v-btn>
-              <v-btn flat icon>
-                <v-icon>edit</v-icon>
-              </v-btn>
-            </v-layout>
-            <v-layout wrap>
-              <v-flex class="xs6">
-                Organism:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.organism.name }}
-              </v-flex>
-              <v-flex class="xs6">
-                Model:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.model.name }}
-              </v-flex>
-              <v-flex class="xs6">
-                Method:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.method }}
-              </v-flex>
-              <v-flex class="xs6">
-                Growth rate:
-              </v-flex>
-              <v-flex class="xs6 text-xs-right">
-                {{ card.growthRate }} <em>h<sup>-1</sup></em>
-              </v-flex>
-            </v-layout>
-          </v-card-title>
-        </v-card>
+        <Card
+          v-for="card in cards"
+          :key="cards.indexOf(card)"
+          :card="card"
+          :isLastCard="cards.length === 1"
```

Comment:
This tests if it's the only card. Should the test be `index === cards.length - 1`?

Reply:
I see now. Can you rename this variable to `isOnlyCard` maybe?